python/py-fastdtw/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    py-fastdtw
+version                 0.3.4
+categories-append       science
+platforms               darwin
+license                 MIT
+maintainers             nomaintainer
+description             Dynamic Time Warping (DTW) algorithm with an O(N) time and memory complexity.
+long_description        Python implementation of FastDTW, which is an approximate Dynamic Time Warping (DTW) algorithm that provides optimal or near-optimal alignments with an O(N) time and memory complexity.
+
+distname                ${python.rootname}-${version}
+master_sites            pypi:[string index ${python.rootname} 0]/${python.rootname}/
+homepage                https://pypi.python.org/pypi/${python.rootname}/
+
+checksums               rmd160  a8f9469f045a944f0575e2c54f4880881397510c \
+                        sha256  2350fa6ec36bcad186eaf81f46eff35181baf04e324f522de8aeb43d0243f64f \
+                        size    133402
+
+python.versions         37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append  port:py${python.version}-setuptools \
+                          port:py${python.version}-requests
+
+    depends_lib-append    port:py${python.version}-numpy
+
+    livecheck.type        none
+} else {
```

Comment:
The `python` portgroup should automatically be taking care of the `livecheck` details, so you really should only need the `if` part of this, and not the `else`.  Am I mistaken?

Reply:
Thank you for your review.
I have removed the `else` block, squashed the commit, and pushed as 3cf45cd .
Please have a look.